pkgs/applications/version-management/gitlab/update-pg.patch

```diff
@@ -1,13 +1,13 @@
 diff --git a/Gemfile.lock b/Gemfile.lock
-index f04445e1d..b9bf34cd0 100644
+index 9074b48c56b..57a1c924897 100644
 --- a/Gemfile.lock
 +++ b/Gemfile.lock
-@@ -987,7 +987,7 @@ GEM
+@@ -1019,7 +1019,7 @@ GEM
        tty-color (~> 0.5)
      peek (1.1.0)
        railties (>= 4.0.0)
 -    pg (1.4.1)
 +    pg (1.4.3)
-     pg_query (2.1.3)
+     pg_query (2.1.4)
```

Comment:
Did this fail to cleanly apply because of this line (`pg_query`)? Seems weird since it only touches the `pg` line, which hasn't changed, but just making sure. (I ask this since I don't see the point in updating the patch if it still cleanly applies as expected.)

I really should just open an issue or an MR for this to get it fixed in the next release, who knows if it'll get lost in their sea of issues/MRs, though.

Reply:
> Did this fail to cleanly apply because of this line (pg_query)?

Exactly.